.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+# Golang CircleCI 2.1 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:latest
```

Comment:
Might I recommend you also add the specific version tag for today's :latest? That way it stays covered even after the docker image is updates!